stdlib/proposals/ai/agent_identity_in_bi.md

```
@@ -0,0 +1,676 @@
+
+   # Optional authentication details of the agent.
+   @display {label: "Agent Credential"}
+   AgentCredential agentCredential?;
+|};
+```
+
+#### Agent Credential
+
+A new configuration is introduced to add the agent’s credentials.
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)